backend/controllers/WebhookController.ts

```diff
@@ -2,16 +2,16 @@ import { Express, Request, Response } from "express";
 import { BaseController } from "./BaseController";
 import { HttpStatusCode } from "axios";
 
-export class  WebhookController extends BaseController {
-    constructor(app: Express) {
-        super(app, '/webhooks');
-    }
+export class WebhookController extends BaseController {
+  constructor(app: Express) {
+    super(app, "/webhooks");
```

Comment:
nit: this repo needs a prettier rule for `single-quote` or `double-quote`, currently it's not consistent.

Reply:
`single-quote` seems to be our standard in other repos